app/src/main/res/xml/prefs.xml

```diff
@@ -1,75 +1,90 @@
 <?xml version="1.0" encoding="utf-8"?>
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There's no need to add the `androidx.preference.` prefix to any elements in this file.

Reply:
How does the compiler figure out whether it should use `android.preference.PreferenceScreen` or `androidx.preference.PreferenceScreen` if I omit the namespace?

Reply:
You're assuming this is checked at compile time. You can easily test this assumption by e.g. adding `<Testing/>` to the file and building the app.

<details>
<summary>Spoiler</summary>
To the build tools it's just an XML file. At runtime the inflater expands the tag names to fully qualified class names if necessary and tries to load the classes. You indirectly select the inflater that is used by using either `android.preference.PreferenceFragment` or `androidx.preference.PreferenceFragmentCompat`.
</details>

Reply:
:heavy_check_mark: Removed namespace in 8a1cdab07ba57e017e94c5ead9cb6d2309773bc2. Thank you.